app/controllers/v1/audits_controller.rb

```diff
@@ -22,11 +22,8 @@ def create
       audit = Audit.new(user: current_user, packages: packages.count)
 
       resource_policy.new(current_user, audit).authorize! :save?
-      audit.save
 
-      packages.each do |package|
-        AuditFixityCheckJob.perform_later(package: package, user: current_user, audit: audit)
-      end
+      AuditManager.start_audit (user: current_user, audit: audit, packages: packages)
```

Comment:
We usually need to squish the parentheses and method name together... It's a weird Ruby parsing thing.